micro_dl/utils/io_utils.py

```diff
@@ -46,9 +46,21 @@ def write_meta_field(position: ngff.Position, metadata, field_name):
     :param str field_name: name of new/existing field to write to
     """
     if field_name in position.zattrs:
-        position.zattrs[field_name].update(metadata)
+        if subfield_name in position.zattrs[field_name]:
+            position.zattrs[field_name][subfield_name].update(metadata)
+        else:
+            D1 = position.zattrs[field_name]
+            field_metadata = {
+                subfield_name: metadata,
+            }
+            # position.zattrs[field_name][subfield_name] = metadata
```

Comment:
I'm not sure if I read this correctly: is this avoiding overwriting existing `subfield_name` and attempting to duplicate the key?

Reply:
Also if at one level there will always be only one `subfield_name` why is it necessary?

Reply:
I have deleted the `io_utils.py` file and transferred the two functions to `meta_utils.py` file. The functions are also called in `mp_utils.py` file, so I have changed it there as well.

I have edited the docstring as well.